portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -58,6 +69,13 @@ private String convertToJson(List<String> data) {
     return new Gson().toJson(data);
   }
 
+  /**
+   * @return the desired comment format to display
+   */
+  private String getCommentStatement(String name, String comment, long timestamp) {
+    return "at " + timestamp + ", "  + name + " said: " + comment;
```

Comment:
prefer String.format.

And it's not required to have a java doc for private method.

Reply:
Done.